app/src/views/GuestApplicationTracker.tsx

```diff
@@ -1,5 +1,97 @@
 import {Divider, Box, Typography, Stack, useTheme} from '@mui/material';
 import {styled} from '@mui/system';
+import {TaskAccordion} from '../components/dashbaord/DashboardTaskAccordion';
+
+export type TaskStatus = 'in-progress' | 'complete' | 'locked';
+
+export type TaskIds =
+  | 'application_and_onboarding'
+  | 'host_matching'
+  | 'host_match_finalization';
+
+export type SubTaskIds =
+  | 'submit_application'
+  | 'interview_with_coordinator'
+  | 'attend_training_session'
+  | 'match_with_host'
+  | 'initial_host_meeting'
+  | 'sign_agreement';
+
+export interface SubTask {
+  id: SubTaskIds;
+  title: string;
+  status: TaskStatus;
+  description: string;
+}
+
+export interface Task {
+  id: TaskIds;
+  title: string;
+  status: TaskStatus;
+  subTasks: SubTask[];
+}
+
+const tasks: Task[] = [
+  {
+    id: 'application_and_onboarding',
+    title: 'Application and Onboarding',
+    status: 'in-progress',
+    subTasks: [
```

Comment:
Do the subtasks need a link or other data to let the FE know where to send the user to complete the subtask?

Reply:
@paulespinosa Yes, if that's something we could keep in the db that would be great.